src/pages/LendPage/LendPage.tsx

```diff
@@ -0,0 +1,435 @@
+          DeFi.
+        </Box>
+      </Box>
+      <Box mt={'48px'} display={'flex'} gridGap={'24px'} flexWrap={'wrap'}>
+        <Box
+          flex={'1'}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+          sx={{ minWidth: { xs: '35%', sm: '35%', md: '20%' } }}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Supply
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+          sx={{ minWidth: { xs: '35%', sm: '35%', md: '20%' } }}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Total Borrowed
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+          sx={{ minWidth: { xs: '35%', sm: '35%', md: '20%' } }}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Liquidity
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+        <Box
+          flex={'1'}
+          bgcolor={'#232734'}
+          p={'24px'}
+          borderRadius={'12px'}
+          sx={{ minWidth: { xs: '35%', sm: '35%', md: '20%' } }}
+        >
+          <Box color={'#696c80'} fontSize={'14px'}>
+            Assets
+          </Box>
+          <Box fontSize={'24px'} color={'white'}>
+            $19.2M
+          </Box>
+        </Box>
+      </Box>
+      <Box
+        my={'32px'}
+        display={'flex'}
+        alignItems={'center'}
+        sx={{
+          justifyContent: { xs: 'center', sm: 'center', md: 'space-between' },
+        }}
+        gridGap={'24px'}
+        flexWrap={'wrap'}
+      >
+        <Box display={'flex'} gridGap={'24px'}>
+          <Box fontSize={'18px'} fontWeight={'bold'} color={'#448aff'}>
+            All Pools
+          </Box>
+          <Box fontSize={'18px'} fontWeight={'bold'} color={'#626680'}>
+            My Pools
+          </Box>
+        </Box>
+        <Box
+          maxWidth={'100%'}
+          display={'flex'}
+          justifyContent={'center'}
+          alignItems={'center'}
+          gridGap={'16px'}
+        >
+          <SearchInput
+            width={'calc(50% - 8px)'}
+            placeholder={
+              // isMobile ? 'Search' : 'Search name, symbol or paste address'
+              'Search'
+            }
+            value={searchInput}
+            setValue={setSearchInput}
+          />
+          <CustomSelect width={'calc(50% - 8px)'}>
+            <SmOption>SortBy: Rewards</SmOption>
+            <SmOption>12313123123</SmOption>
+            <SmOption>12313123123</SmOption>

```

Comment:
123123123????

Reply:
fixed the options based on pool names for now.